core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java

```diff
@@ -104,7 +124,7 @@ public void importData() throws Exception
         FileReader reader =  new FileReader(clinicalDataFile);
         BufferedReader buff = new BufferedReader(reader);
         List<ClinicalAttribute> columnAttrs = grabAttrs(buff);
-
+        
```

Comment:
@pieterlukasse  Only the sample id is specific to sample-level files; patient ids are required in both. Patient id is actually the *only* column that can be allowed in both since sample attributes and patient attributes are stored in the same table and cannot have the same name.